### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by Lion-Wei**

### CODE REVIEW

File: contributors/devel/developer-guides/vagrant.md [hunk 1]
```diff
@@ -227,7 +227,7 @@ my-nginx   3         3         3            3           1m
 
 We did not start any Services, hence there are none listed. But we see three
 replicas displayed properly. Check the
-[guestbook](https://github.com/kubernetes/kubernetes/tree/%7B%7Bpage.githubbranch%7D%7D/examples/guestbook)
+[guestbook](https://github.com/kubernetes/kubernetes/tree/master/examples/guestbook)
```
Comment (cblecker) [inline contributors/devel/developer-guides/vagrant.md]: This has moved. New URL: https://git.k8s.io/examples/guestbook

File: contributors/devel/developer-guides/vagrant.md [hunk 2]
```diff
@@ -227,7 +227,7 @@ my-nginx   3         3         3            3           1m
 
 We did not start any Services, hence there are none listed. But we see three
 replicas displayed properly. Check the
-[guestbook](https://github.com/kubernetes/kubernetes/tree/%7B%7Bpage.githubbranch%7D%7D/examples/guestbook)
+[guestbook](https://github.com/kubernetes/examples/tree/master/guestbook)
```
Comment (cblecker) [inline contributors/devel/developer-guides/vagrant.md]: Please actually use this URL: https://git.k8s.io/examples/guestbook

Comment (Lion-Wei) [reply contributors/devel/developer-guides/vagrant.md]: sorry, didn't notice that, fixed.

Comment (cblecker) [reply contributors/devel/developer-guides/vagrant.md]: Still doesn't appear to be updated

Comment (Lion-Wei) [reply contributors/devel/developer-guides/vagrant.md]: Yeah,,, I changed it to the url that https://git.k8s.io/examples/guestbook jump to, or maybe I should changed the url to this one https://kubernetes.io/docs/tutorials/stateless-application/guestbook/ ?

**Pull request merged**